app/services/cocktail_service.rb

```diff
@@ -8,9 +8,22 @@ def self.get_cocktail_details(cocktail_id)
     parse_json(response)
   end
 
-  private
+  def self.search_cocktails(query)
+    response = Faraday.get('https://www.thecocktaildb.com/api/json/v1/1/search.php?') do |req|
+      req.params['s'] = query
+      req.params['api_key'] = ENV['cocktail_key']
+    end
+
+    parse_json(response)
+  end
+
+  # def self.conn
+  #   Faraday.new(url: 'https://www.thecocktaildb.com/api/json/v1/1')
+  # end
```

Comment:
I'm all for this refactor! -- you can also include the api_key in this method possibly, since we'd otherwise repeat it for every single request. Generally, I'm in favor of adding in all of the params that'll be in every request here.

Reply:
Tried refactoring using the `conn` method, but had some issues with vcr; so I left the method in there to be refactored at a later time... prob can slim it down, per your comment on the api_key.